freud/order.pyx

```diff
@@ -162,20 +162,23 @@ cdef class Cubatic(_Compute):
 cdef class Nematic(_Compute):
     r"""Compute the nematic order parameter for a system of particles.
 
-    Args:
-        u (:math:`\left(3 \right)` :class:`numpy.ndarray`):
-            The nematic director of a single particle in the reference state
-            (without any rotation applied).
+    In some cases orientations are given as quaternions representing a rotation
+    from the starting reference state (orientation) - such as HOOMD-blue. To
+    convert from quaternions to orientations in 3D one should apply quaternions
+    on the initial reference orientation of the particle.
+
+    Example::
+
+        >>> import rowan
+        >>> import numpy as np
+        >>> quats = np.array([[1, 0, 0, 0]] * 100)
+        >>> starting_orientation = np.array([1, 1, 0])
+        >>> orientations = rowan.rotate(quats,starting_orientation)
```

Comment:
Let's add two more lines to this example showing how orientations are given to the `Nematic` class's compute method.